DatePicker.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  DatePicker.swift
+//  BlueprintUICommonControls
+//
+//  Created by Kyle Bashour on 4/22/22.
+//
+
+import BlueprintUI
+
+public struct DatePicker: Element {
+
+    public var date: Date?
+
+    public var minimumDate: Date?
+
+    public var maximumDate: Date?
+
+    public var datePickerMode: UIDatePicker.Mode = .dateAndTime
+
+    private var _preferredDatePickerStyle: Any?
+
+    @available(iOS 13.4, *)
+    public var preferredDatePickerStyle: UIDatePickerStyle {
+        get { _preferredDatePickerStyle as? UIDatePickerStyle ?? .automatic }
+        set { _preferredDatePickerStyle = newValue }
+    }
+
+    public var onChange: (Date) -> Void
+
+    public init(
+        date: Date?,
+        onChange: @escaping (Date) -> Void,
+        configure: ((inout DatePicker) -> Void)? = nil
+    ) {
+        self.date = date
+        self.onChange = onChange
+        configure?(&self)
+    }
+
+    public var content: ElementContent {
+        struct Measurer {
```

Comment:
Do you need this; can you just do it inline in the measure block?

Reply:
Need a place for the static instance, so it's nicely scoped with this type.